slow_benchmarks/matrices.py

```diff
@@ -0,0 +1,455 @@
+from sympy import Symbol, Matrix, symbols, SparseMatrix, Abs
+from sympy import I, S, simplify
+from sympy.simplify.simplify import simplify
+
+_TEST_SIMPLIFY = False # test simplify after operation?

```

Comment:
Why is this needed?

Reply:
In the original benchmark since dotprodsimp was optional this was for comparing `func(dotprodsimp=True)` vs. `simplify(func())`. I left the code in just in case at some point someone wants to make the same comparison.